apps/emqx_retainer/src/emqx_retainer_mnesia.erl

```diff
@@ -48,12 +48,16 @@ create_resource(#{storage_type := StorageType}) ->
                  disc      -> disc_copies;
                  disc_only -> disc_only_copies
              end,
+    TableType = case Copies of
+                    disc_only -> set;
```

Comment:
Curiosity: we want to use `set` for `disc_only` for the case that this table already exists in disk with this (original) type, right?

Reply:
no. dets has no ordered_set option.